lib/utils/tests-options-validator.js

```diff
@@ -48,7 +48,49 @@ Object.defineProperty(TestsOptionsValidator.prototype, 'shouldSplit', {
  */
 function validatePartitions(partitions, split) {
   validatePartitionSplit(partitions, split);
-  validatePartitionsUnique(partitions);
+  validateElementsUnique(partitions);
+}
+
+/**
+ * Validates the specified reply-browser
```

Comment:
replay-browser, not reply-browser ('reply' shows up in several places, and I assume should be changed to 'replay' in all of them?)